Makefile

```diff
@@ -24,5 +24,9 @@ nose3: qless-core
 	rm -rf .coverage
 	nosetests3 --exe --cover-package=qless --with-coverage --cover-branches -v
 
+.PHONY: requirements
+requirements:
```

Comment:
Now that you mention it, since `make` has no way of telling when the dependencies has changed, you do probably want this to be a phony target, so that you can regenerate it based on the current state by hand.